package.json

```diff
@@ -30,7 +30,7 @@
     "@types/jasmine": "^3.3.12",
     "@types/lowdb": "^1.0.5",
     "@types/lunr": "^2.3.3",
-    "@types/node": "^10.9.4",
+    "@types/node": "^12.12.54",
```

Comment:
We still use node 10 for building our apps. Is this necessary?

Reply:
We get compile errors otherwise, I can try using a newer version of 10.

Reply:
10.17.28 seems to work